README.md

```diff
@@ -38,7 +38,11 @@ You can view the generated [reference docs][javadoc] for a full list of classes
 
 ```groovy
 dependencies {
+    // Utilities for Maps SDK for Android
     implementation 'com.google.maps.android:android-maps-utils:1.2.1'
+
+    // Utilities for Maps SDK for Android V3 BETA
```

Comment:
I'd suggest some indication here that both dependencies shouldn't be used together - that may not be obvious to those unfamiliar with the SDK.

```suggestion
    // Alternately - Utilities for Maps SDK for Android V3 BETA
```

Reply:
Good idea. A similar edit should be made in the KTX lib.